chiselFrontend/src/main/scala/chisel3/core/Module.scala

```diff
@@ -113,7 +113,7 @@ extends HasId {
     Port(iodef)
   }
 
-  private[core] val _namespace = Builder.globalNamespace.child
+  private[core] val _namespace = Namespace.empty
```

Comment:
Add comment noting that in Firrtl, the global and instance namespaces are disjoint